package.json

```diff
@@ -7,11 +7,12 @@
     "npm": "10.2.4"
   },
   "scripts": {
-    "build": "cross-env NODE_ENV=production concurrently --kill-others-on-fail npm:build:*",
+    "build": "npm run generate-documentation && cross-env NODE_ENV=production concurrently --kill-others-on-fail npm:build:*",
```

Comment:
Is this called only from CI during release?

I want to make sure we are mindful of calling this because it will eventually call into OpenAI, and may cost money.

Reply:
> Is this called only from CI during release?

We can configure that from the config files sir. Once done with proper testing I will disable this for preview env.